assignment/scripts/part-supply.js

```diff
@@ -32,10 +32,14 @@ console.log('---  Stretch Goals  ---');
 // 7. Rewrite the `for` loop from #6 as a `for of` loop. 
 console.log('7. Showing supplyChanges with "for of" loop');
 
-// 8. Rewrite the `for` loop from #6 as a `while` loop.
-console.log('8. Showing supplyChanges with "while" loop');
-
-
-// 9. Write a loop to determine the total number of parts available by
+// 8. Write a loop to determine the total number of parts available by
```

Comment:
Swapped 8 with 9. This _total parts_ question is still dealing with the `supplyChanges` array. The boxes question is not.